<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersPanel</class>
 <widget class="QWidget" name="ParametersPanel">
  <property name="windowTitle">
   <string>Simulation Parameters</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>8</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>

   <!-- Grid Group -->
   <item>
    <widget class="QGroupBox" name="gridGroupBox">
     <property name="title">
      <string>Grid</string>
     </property>
     <layout class="QFormLayout" name="gridFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nxLabel">
        <property name="text">
         <string>NX</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="nxSpinBox">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="nyLabel">
        <property name="text">
         <string>NY</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="nySpinBox">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>500</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Simulation Group -->
   <item>
    <widget class="QGroupBox" name="simulationGroupBox">
     <property name="title">
      <string>Simulation</string>
     </property>
     <layout class="QFormLayout" name="simulationFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="timeStepsLabel">
        <property name="text">
         <string>Time Steps</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="timeStepsSpinBox">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="value">
         <number>1000</number>
        </property>
       </widget>
      </item>

      <item row="1" column="0">
       <widget class="QLabel" name="diffusionLabel">
        <property name="text">
         <string>Diffusion Coeff.</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="diffusionSpinBox">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.0</double>
        </property>
        <property name="maximum">
         <double>1.0</double>
        </property>
        <property name="singleStep">
         <double>0.01</double>
        </property>
        <property name="value">
         <double>0.1</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Run Button -->
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run Simulation</string>
     </property>
     <property name="minimumHeight">
      <number>36</number>
     </property>
    </widget>
   </item>

   <!-- Spacer -->
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
